example/libbpftime_example/spdlog_exam.cpp

```diff
@@ -0,0 +1,5 @@
+#include "spdlog/spdlog.h"
+
+int main() {
```

Comment:
I think maybe you can remove this example, or replace it with one that exec the eBPF bytecode, like in tools

Reply:
ok, will check this out.

Reply:
I have removed this one. First one is enough to showcase usage of `libbpftime.a`